config/application.rb

```diff
@@ -8,7 +8,8 @@
 require 'govuk/components'
 # Require the gems listed in Gemfile, including any gems
 # you've limited to :test, :development, or :production.
-Bundler.require(*Rails.groups)
+groups = Rails.env[/_aks/] ? Rails.groups + [:production] : Rails.groups
```

Comment:
This is a way to load the production gems when the `RAILS_ENV` is set to `_aks`.
Another approach would be to declare a variable in the Gemfile and splat it out as an argument to `group` method. But this requires code change if another `*_aks` is added.

```ruby
aks_envs = %i[production_aks qa_aks sandbox_aks review_aks staging_aks]

group(*aks_envs, :production, :qa, :sandbox, :staging) do
  gem ...
end
```